app/composables/useDataHover.test.ts

```diff
@@ -82,4 +82,46 @@ describe('useDataHover', () => {
     setHover('Hit', '✅', true)
     expect(getDelta(5)).toBe(null)
   })
+
+  it('should compare Dates using getTime', () => {
```

Comment:
this doesn't describe the intended behaviour, it describes some arbitrary implementation detail